RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -10,155 +10,155 @@
-  process.TrackstersMIP = trackstersProducer.clone(
-      filtered_mask = cms.InputTag("FilteredLayerClustersMIP", "MIP"),
-      seeding_regions = cms.InputTag("SeedingGlobal"),
+  process.trackstersMIP = trackstersProducer.clone(
+      filtered_mask = cms.InputTag("filteredLayerClustersMIP", "MIP"),
+      seeding_regions = cms.InputTag("seedingGlobal"),
       missing_layers = 3,
       min_clusters_per_ntuplet = 15,
       min_cos_theta = 0.99, # ~10 degrees
       min_cos_pointing = 0.9
   )
 
-  process.MultiClustersFromTrackstersMIP = multiClustersFromTrackstersProducer.clone(
+  process.multiClustersFromTrackstersMIP = multiClustersFromTrackstersProducer.clone(
       label = "MIPMultiClustersFromTracksterByCA",
-      Tracksters = "TrackstersMIP"
+      Tracksters = "trackstersMIP"
   )
 
-  process.FilteredLayerClusters = filteredLayerClustersProducer.clone(
+  process.filteredLayerClusters = filteredLayerClustersProducer.clone(
       algo_number = 8,
       iteration_label = "algo8",
-      LayerClustersInputMask = "TrackstersMIP"
+      LayerClustersInputMask = "trackstersMIP"
   )
 
-  process.Tracksters = trackstersProducer.clone(
-      original_mask = "TrackstersMIP",
-      filtered_mask = cms.InputTag("FilteredLayerClusters", "algo8"),
-      seeding_regions = cms.InputTag("SeedingGlobal"),
+  process.tracksters = trackstersProducer.clone(
+      original_mask = "trackstersMIP",
+      filtered_mask = cms.InputTag("filteredLayerClusters", "algo8"),
+      seeding_regions = cms.InputTag("seedingGlobal"),
       missing_layers = 2,
       min_clusters_per_ntuplet = 15,
       min_cos_theta = 0.94, # ~20 degrees
       min_cos_pointing = 0.7
   )
 
-  process.MultiClustersFromTracksters = multiClustersFromTrackstersProducer.clone(
-      Tracksters = "Tracksters"
+  process.multiClustersFromTracksters = multiClustersFromTrackstersProducer.clone(
+      Tracksters = "tracksters"
   )
 
   process.hgcalMultiClusters = hgcalMultiClusters
   process.TICL_Task = cms.Task(
-      process.TICLLayerTileProducer,
-      process.SeedingTrk,
-      process.FilteredLayerClustersTrk,
-      process.TrackstersTrk,
-      process.MultiClustersFromTrackstersTrk,
-      process.SeedingGlobal,
-      process.FilteredLayerClustersMIP,
-      process.TrackstersMIP,
-      process.MultiClustersFromTrackstersMIP,
-      process.FilteredLayerClusters,
-      process.Tracksters,
-      process.MultiClustersFromTracksters)
+      process.ticlLayerTileProducer,
+      process.seedingTrk,
+      process.filteredLayerClustersTrk,
+      process.trackstersTrk,
+      process.multiClustersFromTrackstersTrk,
+      process.seedingGlobal,
+      process.filteredLayerClustersMIP,
+      process.trackstersMIP,
+      process.multiClustersFromTrackstersMIP,
+      process.filteredLayerClusters,
+      process.tracksters,
+      process.multiClustersFromTracksters)
   process.schedule.associate(process.TICL_Task)
   return process
 
 def TICL_iterations(process):
```

Comment:
> please add a comment to denote the purpose of this method.

this would help to clarify the purpose of the method.
Thanks for adding a note for the first method

Reply:
>  It seems more practical to merge the two.

is there a good reason to keep this separated?
From https://github.com/cms-sw/cmssw/pull/27439#discussion_r310090538 I somehow (mis?)read that the rest can be the same as well